prime-router/settings/organizations-local.yml

```diff
@@ -544,9 +544,37 @@
         topic: covid-19
```

Comment:
I plan to merge organizations.yml and organizations-local.yml into one file, when I get a chance - I don't think there's a need to keep both.

Reply:
I agree. We need to sync them all up and clean them up.